### DISCUSSION TITLE
Mirror, Mirror, on the Image

### DISCUSSION THREAD

**Discussion by Jay123210599** [ImageMagick/ImageMagick#7375] [Help]
### ImageMagick version

7.1.1-27

### Operating system, version and so on

Windows 11

### Description

Is it possible to mirror my images without losing quality? If so, how?

**Comment by fmw42** [comment]
Mirror what image?  Using -flip or -flop does not degrade the image.

**Reply by Jay123210599** [reply]
What does flip and flop do?

**Reply by fmw42** [reply]
You need to look those up yourself.  See https://imagemagick.org/script/command-line-options.php. Click the selector near the top left of the page where it says "Select CLI Option".  Then scroll to -flip and -flop.  Click on one of them.

(They mirror the image vertically or horizontally.)
